frontend/src/index.html

```diff
@@ -79,6 +79,98 @@
         </div>
       </div>
     </nav>
+
+    <!-- player UI design -->
+    <div class="my-player text-test fixed z-10 bottom-0 w-full bg-white border border-solid border-gray-300 rounded-t-md shadow-upward">
+      <div class="back absolute z-0 w-full h-full"></div>
```

Comment:
What is this empty `div` for? Not clear. Either make it obvious by a comment, or remove.